extras/capitanodoc/index.ts

```diff
@@ -0,0 +1,33 @@
+import capitanodoc = require('../../capitanodoc');
+import * as _ from 'lodash';
+import * as path from 'path';
+import * as markdown from './markdown';
+import { Document, Category } from './doc-types';
+
+const result = <Document> {};
+result.title = capitanodoc.title;
+result.introduction = capitanodoc.introduction;
+result.categories = [];
+
+for (let commandCategory of capitanodoc.categories) {
+	const category = <Category> {};
+	category.title = commandCategory.title;
+	category.commands = [];
+
+	for (let file of commandCategory.files) {
+		const actions: any = require(path.join(process.cwd(), file));
+
+		if (actions.signature != null) {
+			category.commands.push(_.omit(actions, 'action'));
+		} else {
+			for (let actionName in actions) {
```

Comment:
Strangely it didn't seem to mind, maybe because `actions` is `any`? No idea, but I've moved to `for of Object.keys` instead, which is better regardless.